frontend-runitup/src/Components/Routes/RoutesPage.jsx

```diff
@@ -154,12 +161,12 @@ const RoutesPage = () => {
 
     try {
```

Comment:
This try statement is starting to be pretty long. Is it possible to separate it out into functions to make the try statement smaller? The smaller functions can have good, descriptive names that indicate what it's doing overall. Try to group together different calls you're making and separate them into its own function